Esp8266/dummy_client.cpp

```diff
@@ -0,0 +1,64 @@
+#ifndef ARDUINO
```

Comment:
I love this. It could use a *little* documentation?

Reply:
Yes boss. Maybe in the next pr, when I actually start using InputData() and OutputData() the way I had pllanned.